_raml/apps/products/products.raml

```diff
@@ -130,23 +130,51 @@ post:
     responses:
       204:
 
-  /crossselling:
-    displayName: Product crossselling
-    get:
-      is: [ locale, currency ]
-      queryParameters:
-        includeInvisible:
-          description: Includes invisible products. Only available with `products_write` authorisation.
-          type: boolean
-          required: false
-      description: Returns information on the crossselling of products. *epagesNow
-      securedBy: [products_read]
-      responses:
-        200:
-          body:
-            application/json:
-              example: !include examples/get_product_productid_crossselling_response.json
-              schema: !include schema/get_product_productid_crossselling.schema.json
+/crossselling:
+  displayName: Product crossselling
+  get:
+    is: [ locale, currency ]
+    queryParameters:
+      includeInvisible:
+        description: Includes invisible products. Only available with `products_write` authorisation.
+        type: boolean
+        required: false
+    description: Returns information on the crossselling of products. *epagesNow
+    securedBy: [products_read]
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include examples/get_product_productid_crossselling_response.json
+            schema: !include schema/get_product_productid_crossselling.schema.json
+
+  put:
+    description: Replace all cross selling assignments
+    securedBy: [products_write]
+    body:
+      application/json:
+        example: !include examples/put_product_productid_crossselling_request.json
+        schema: !include schema/put_product_productid_crossselling_request_schema.json
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include examples/get_product_productid_crossselling_response.json
+            schema: !include schema/get_product_productid_crossselling.schema.json
```

Comment:
```suggestion
            schema: !include schema/put_product_productid_crossselling.schema.json
```
I think the same thing applies to the schema file.
**Note:** This schema file doesn't exist yet and needs to be created first!

Reply:
Please see above comment

Reply:
I've checked again with other endpoints and it seems that even though it is the same thing for the `GET` request, we still have two files (see screenshot). I know it's duplicated content with a different title but since we've done it like this before, I would like to keep it that way. I guess the same thing applies to the examples. 
![Bildschirm­foto 2023-02-23 um 16 20 39](https://user-images.githubusercontent.com/108728944/220954362-5af8d5a5-b496-408f-859e-83e156ac3baa.png)

Reply:
Updated